src/window/plain_message_dialog.c

```diff
@@ -47,24 +47,27 @@ static void draw_foreground(void)
     graphics_reset_dialog();
 }
 
-static void handle_mouse(const mouse *m)
+static void close(void)
+{
+    window_go_back();
+}
+
+static void handle_input(const mouse *m, const hotkeys *h)
 {
     if (image_buttons_handle_mouse(mouse_in_dialog(m), 80, 80, buttons, 1, 0)) {
         return;
     }
-    if (input_go_back_requested()) {
-        window_plain_message_dialog_accept();
+    if (input_go_back_requested(m, h)) {
```

Comment:
Why not just:

```
if (input_go_back_requested(m, h) || h->enter_pressed) {
    window_go_back();
}
```

Reply:
Probably because I was quite tired after refactoring about 70 screens. I'll simplify this.

Reply:
Probably because I was quite tired after refactoring about 70 screens. I'll simplify this.